Multisig/Data/Services/Safe Relay Service/SafeGelatoRelayService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SafeGelatoRelayService.swift
+//  Multisig
+//
+//  Created by Vitaly on 05.03.23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class SafeGelatoRelayService {
+
+    var url: URL
+
+    private let httpClient: JSONHTTPClient
+    private let mockHttpClient: MockJSONHttpClient
+
+    var jsonDecoder: JSONDecoder {
+        httpClient.jsonDecoder
+    }
+
+    init(url: URL, logger: Logger) {
+        self.url = url
+        httpClient = JSONHTTPClient(url: url, logger: logger)
+        httpClient.jsonDecoder.dateDecodingStrategy = .millisecondsSince1970
+        mockHttpClient = MockJSONHttpClient()
+    }
+
+    @discardableResult
+    func execute<T: JSONRequest>(request: T) throws -> T.ResponseType {
+        try httpClient.execute(request: request)
+    }
+
+    func asyncExecute<T: JSONRequest>(request: T, completion: @escaping (Result<T.ResponseType, Error>) -> Void) -> URLSessionTask? {
+        httpClient.asyncExecute(request: request, completion: completion)
+    }
+
+    // Returns mocked response in completion
+    func asyncExecuteMock<T: JSONRequest>(request: T, completion: @escaping (Result<T.ResponseType, Error>) -> Void) -> URLSessionTask? {
```

Comment:
Why do we need this?

Reply:
Can we use it to have a unit test that returns a predefined response? For example to test that the number in `expiresAt` is converted to a Date?

Reply:
this was the idea behind it. Unfortunately we are not using MockJSONHttpClient in our tests yet.